test/unit/marked-spec.js

```diff
@@ -1,7 +1,21 @@
-var marked = require('../../lib/marked.js');
+var sut = require('../../lib/marked.js');
 
 it('should run the test', function () {
-  spyOn(marked, 'parse').and.callThrough();
-  marked.parse('Hello World!');
-  expect(marked.parse).toHaveBeenCalled();
+  spyOn(sut, 'parse').and.callThrough();
+  sut.parse('Hello World!');
+  expect(sut.parse).toHaveBeenCalled();
+});
+
+describe('Test heading ID functionality', function() {
+	it('should add id attribute by default', function() {
+		var renderer = new sut.Renderer(sut.defaults);
```

Comment:
Could we change `sut` to something like `markedUnderTest` or something like that? It feels very unintuitive that these defaults are coming from marked.

I feel like the only reason to distinguish the "system under test" is if we would need the same system **not** under test. (e.g. Jasmine testing Jasmine with Jasmine) I can't think of a reason we would ever need that.